Specs/DateTime/Korean/DateParser.json

```diff
@@ -1919,38 +1879,14 @@
             "date": "1989-05-05"
           }
         },
-        "Start": 13,
-        "Length": 8
-      }
-    ]
-  },
-  {
-    "Input": "2071년 3월 7일에 돌아갈 거야. ",
-    "Context": {
-      "ReferenceDateTime": "2016-11-07T00:00:00"
-    },
-    "NotSupported": "dotnet",
-    "NotSupportedByDesign": "javascript,python,java",
-    "Results": [
-      {
-        "Text": "2017년 3월 7일",
-        "Type": "date",
-        "Value": {
-          "Timex": "1971-05-05",
-          "FutureResolution": {
-            "date": "1971-05-05"
-          },
-          "PastResolution": {
-            "date": "1971-05-05"
-          }
-        },
-        "Start": 13,
-        "Length": 8
+        "Start": 0,
+        "Length": 11
       }
     ]
   },
   {
     "Input": "너는 지금부터 한 달에 두 번씩 일요일마다 시간 괜찮니? ",
+    "Comment": "Not a proper sentence in Korean and it is not possible to give a better translation of the original case 'Are you available two sundays from now?'",
```

Comment:
In these two cases, please delete the invalid specs.

Reply:
will do